Bottom: 6cd89b5de76c63a597906aafa913d94cf57ddde4
Top:    6ad8074427c83efb016120fe4f6c35a21f96f423
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-05-07 17:21:09 +0530

Refresh of debug-bits-v2

---

diff --git a/kernel/sched/idle.c b/kernel/sched/idle.c
index 4e2baec36cc7..fc9d19afdd88 100644
--- a/kernel/sched/idle.c
+++ b/kernel/sched/idle.c
@@ -241,7 +241,7 @@ static void do_idle(void)
 			trace_printk("cpu-%d pm_disabled=%d\n",cpu,pm_disabled);
 	}
 	if (sched_feat(MY_RESET))
-		atomic_set(&per_cpu(nr_lat_sensitive, cpu), 0);
+		per_cpu(nr_lat_sensitive, cpu) = 0;
 	/*
 	 * If the arch has a polling bit, we maintain an invariant:
 	 *
